ukat/mapping/tests/test_t1.py

```diff
@@ -246,22 +246,31 @@ def test_tss_valid_axis(self):
                         inversion_list=np.linspace(0, 2000, 10),
                         affine=self.affine, tss=1, tss_axis=2)
 
+    def test_molli_2p_warning(self):
+        with pytest.warns(UserWarning):
+            mapper = T1(pixel_array=np.zeros((5, 5, 5)),
+                        inversion_list=np.linspace(0, 2000, 5),
+                        affine=self.affine, parameters=2, molli=True)
+
     def test_real_data(self):
         # Get test data
         magnitude, phase, affine, ti, tss = fetch.t1_philips(2)
+        image_molli, affine_molli, ti_molli = fetch.t1_molli_philips()
 
         # Convert times to ms
         ti = np.array(ti) * 1000
         tss *= 1000
+        ti_molli *= 1000
 
         # Crop to reduce runtime
         magnitude = magnitude[37:55, 65:85, :2, :]
+        image_molli = image_molli[37:55, 65:85, :2, :]
 
         # Gold standard statistics
         gold_standard_2p = [1041.581031, 430.129308, 241.512336, 2603.911794]
-        gold_standard_3p = [1405.617672, 596.71554, 281.406566, 3759.175897]
-        gold_standard_3p_single = [1329.543147, 582.270541, 281.406569,
-                                   3334.1764436]
+        gold_standard_3p = [1416.989523, 722.097507, 0.0, 4909.693108]
```

Comment:
Why are `gold_standard_3p` and `gold_standard_3p_single` different to previously? Is it because the of the bounds in `t1.py` being different?

Reply:
Yeah, different bounds. I felt the lower bound of 1.5 for the inversion efficiency was a bit tight and the sample MOLLI data had some quite high T1s (outside) the kidney that were getting clipped so made sense to increase the maximum T1 too.